doc/install/install_via_centos7.md

```diff
@@ -13,18 +13,28 @@ We need your help to keep this up to date by:
 1. Reporting on the [forum](https://forums.2600hz.com/forums/) about where in the guide things broke down, or that you didn't understand.
 2. Using the [edit link](https://github.com/2600hz/sysadmin/edit/master/doc/install/install_via_centos7.md) to edit this document and issue a pull request.
 
+## Gain access to the FreeSWITCH repositories
+
+SignalWire Personal Access Tokens (PAT)s are required to access FreeSWITCH install packages. Follow the [guide on FreeSWITCH's site](https://developer.signalwire.com/freeswitch/FreeSWITCH-Explained/Installation/HOWTO-Create-a-SignalWire-Personal-Access-Token_67240087/#attachments) to generate a token.
+
 ## Setup the server
 
 This guide builds a server using the [CentOS 7 Minimal ISO](http://isoredirect.centos.org/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1804.iso). Once you have that installed on a server (or virtual machine), it is time to setup the host. Some of the commands below are optional (and noted as such) - check whether you need to run them first.
 
-!!! note
-    Please remember to replace '172.16.17.18' with your server's actual IP address (not localhost either). Please remember to replace 'aio.kazoo.com' with your actual Host Name(FQDN).
+> **Warning**
+> 
+> Please remember to replace
```

Comment:
Why replace the admonition? It is rendered on the docs site as a hopefully obvious section to read: https://docs.2600hz.com/sysadmin/doc/install/install_via_centos7/#setup-the-server

Reply:
Ah, I was using the GitHub syntax, I can put it back next week after I return to work from leave